spec.html

```diff
@@ -2713,6 +2713,7 @@ <h1>The Object Type</h1>
       </emu-note>
       <p>Property keys are used to access properties and their values. There are two kinds of access for properties: <em>get</em> and <em>set</em>, corresponding to value retrieval and assignment, respectively. The properties accessible via get and set access includes both <em>own properties</em> that are a direct part of an object and <em>inherited properties</em> which are provided by another associated object via a property inheritance relationship. Inherited properties may be either own or inherited properties of the associated object. Each own property of an object must each have a key value that is distinct from the key values of the other own properties of that object.</p>
       <p>All objects are logically collections of properties, but there are multiple forms of objects that differ in their semantics for accessing and manipulating their properties. Please see <emu-xref href="#sec-object-internal-methods-and-internal-slots"></emu-xref> for definitions of the multiple forms of objects.</p>
+      <p>In addition, some objects are callable; these are referred to as functions or function objects and are described further below. All functions in ECMAScript are members of the Object type.</p>
```

Comment:
instead of "and are described further below", could "function objects" or "functions" link to the below/

Reply:
"function objects" does link (it's a `dfn`'d term), but I want the prose as well so that a reader I think.

Reply:
> so that a reader I think

?

Reply:
lol whoops.

I want the prose so that a reader knows it will be covered in this section without having to click the link, I think.